src/current/v25.2/create-and-configure-changefeeds.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: Create and Configure Changefeeds
-summary: Create and configure a changefeed job for Core and Enterprise.
+summary: Create and configure a changefeed emitting to a sink or a sinkless changefeed.
 toc: true
 docs_area: stream_data
 ---
 
-Core and {{ site.data.products.enterprise }} changefeeds offer different levels of configurability. {{ site.data.products.enterprise }} changefeeds allow for active changefeed jobs to be [paused](#pause), [resumed](#resume), and [canceled](#cancel).
+Changefeeds offer different levels of configurability. Changefeeds emitting to a sink allow for active changefeed jobs to be [paused](#pause), [resumed](#resume), and [canceled](#cancel). Sinkless changefeeds streams changes directly to the SQL session.
```

Comment:
nit: double plural, i think this could read "Sinkless changefeeds _stream_ changes ..."